gnomad/resources/grch38/reference_data.py

```diff
@@ -154,8 +154,22 @@ def _import_dbsnp(**kwargs) -> hl.Table:
 )
 
 dbsnp = VersionedTableResource(
-    default_version="b154",
+    default_version="b156",
     versions={
+        "b156": GnomadPublicTableResource(
+            # can't find this path anymore, is it temporary for intermediate files?
```

Comment:
Yeah it's temp for where It gets written to on import, then it gets synced to the GCP bucket. By default `GnomadPublicTableResource` will use the GCP bucket for loading.
```suggestion
```

Reply:
I removed this comment.